packages/babel-preset-gatsby/src/index.js

```diff
@@ -20,7 +20,7 @@ function preset(context, options = {}) {
   const stage = process.env.GATSBY_BUILD_STAGE || `test`
 
   if (!targets) {
-    if (stage === `build-html`) {
+    if (stage === `build-html` || stage === `test`) {
```

Comment:
Think this makes sense, right? Should transpile to whatever the current version of node is in the unit test runner.